lanGhost.py

```diff
@@ -7,24 +7,10 @@
     import logging
     logging.getLogger("scapy.runtime").setLevel(logging.ERROR)  # Shut up scapy!
 
+    import urllib.request, urllib.parse, netifaces, traceback, threading, telegram, requests, sqlite3, base64, socket, time, nmap, json, sys, os
```

Comment:
Thanks Kama, I will merge the first commit but I like the way the imports look so please revert that back to the original.